### ISSUE TITLE
Finish rimble-ui integration

### ISSUE THREAD

**Issue by TimDaub** [austintgriffith/burner-wallet#204]
## Scope
- The [rim branch](https://github.com/austintgriffith/burner-wallet/tree/rim) of the burner-wallet is actually quite awesome. It relies on Consensys' rimble-ui and allows users to whitelabel the burner-wallet simply by editing a [theme.js](https://github.com/austintgriffith/burner-wallet/blob/rim/src/theme.js) file
- The rim branch is almost ready to be merged into master, but still has a few components not yet migrated

## Deliverables
- Identify all components in the rim branch that are not yet using rimble-ui and make them start using rimble-ui
- Adjust theme.js such that it looks like xdai.io currently looks (orange styled button etc.)

## Gains for the project

- Extremely simplified whitelabeling functionality for the burner-wallet
- In the future, we could even go as far as having contributors make themes for the burner-wallet by switching out theme.js

## Roles
bounty gardener: @TimDaub / 10%
bounty worker: name / 75%
bounty reviewer: name / 15%

**Comment by MaxStalker** [comment]
Hey @TimDaub I would like to tackle this one ;)

**Comment by yanyishuai** [comment]
Integrated PR for #204, #210, #213 — rimble theme/MainCard, LocalSigner key lifecycle, and burner-core send/sign paths.

**Comment by yanyishuai** [comment]
Updated PR #304 — Receive and SendToAddress now use rimble-ui (QR/PublicAddress, Field/Input/Button). Send error callback from #157 preserved.

**Comment by yanyishuai** [comment]
Update on #204 rimble migration (PR #304 branch fix/issue-204-210-213-burner-core-rimble):

- SendWithLink - rimble Field/Input/Button
- CashOut - rimble Field/Input/Button/Icon (Wyre off-ramp flow unchanged)
- Exchange - DAI/ETH/xDai send panels + ETH/DAI toggle migrated to rimble Box/Field/Flex/Button/OutlineButton

Bootstrap remains on bridge/uniswap sub-flows inside Exchange; send panels and cash-out are migrated.

Wallet: Do4v7foHJvRJLpRRoGaVPWX6DDEjX3yTK7J91gpwUQpE

**Comment by yanyishuai** [comment]
Update on #204 — Exchange bridge/Wyre rimble migration (PR #304):

- DAI<->xDai bridge: toggle buttons, amount inputs, max/cancel/send rows
- ETH<->DAI Uniswap: deposit/withdraw forms + progress bars
- Vendor token (xDai<->ERC20): deposit/withdraw rows
- Wyre on-ramp: slider + Buy button (blue rimble Button)
- xDai->ETH gas top-up banner

Bootstrap grid on asset header rows only; all bridge/Wyre action controls are rimble.

Wallet: Do4v7foHJvRJLpRRoGaVPWX6DDEjX3yTK7J91gpwUQpE

**Comment by yanyishuai** [comment]
Working on #204 — xDai orange rimble theme + MainCard migrated to rimble-ui buttons.

**Comment by rovidev95** [comment]
Hi — RoviDev (@rovidev95). Still interested in picking this up if the rim branch merge is still planned. Can confirm scope on remaining rimble-ui components and submit a PR. Algora/Stripe payout ready if there's an active escrow.

**Comment by rovidev95** [comment]
@TimDaub — PR #309 is ready on the **rim** branch (mergeable, clean).

**Scope vs other open PRs (#304/#308 on master):**
- Migrates **all remaining legacy components** to rimble-ui: Exchange, History, Vendor/Admin/Vendors, Advanced, WithdrawFromPrivate, etc.
- **Removes Bootstrap** from package.json + index.js (issue deliverable: drop bootstrap dependency)
- xDAI orange theme primary (\#FF6700\) in theme.js

Happy to rebase or split if you prefer landing MainCard/App changes first. This PR targets \
im\ as described in the issue.

— RoviDev (@rovidev95)

**Comment by rovidev95** [comment]
Single PR for review: **#311** (includes full rimble-ui scope + /claim #204 rovidev95@gmail.com). Closed #309/#310 to avoid duplicate review. @TimDaub

**Comment by rovidev95** [comment]
Split for faster review → **#315** (rimble-ui only). /claim rovidev95@gmail.com in PR body. Series: #313 #314 #315 #316 #317 #318 @TimDaub

**Comment by rovidev95** [comment]
Split PR for incremental review: **#315** — /claim rovidev95@gmail.com in PR body. @TimDaub

**Comment by stmr** [comment]
I can take this if still available/payable.

Planned scope: Finish rimble-ui integration.

I will keep the PR/submission focused, include tests/docs where requested, run project checks, and wait for maintainer or bounty-platform confirmation before starting work.
